docs/features/software-templates/writing-templates.md

```diff
@@ -801,34 +801,33 @@ The `projectSlug` filter generates a project slug from a repository URL
 - **Input**: `github.com?repo=backstage&org=backstage`
 - **Output**: `backstage/backstage`
 
-## Custom Filters
+## Custom Filters and Globals
 
-Whenever it is needed to extend the built-in filters with yours `${{ parameters.name | my-filter1 | my-filter2 | etc }}`, then you can add them
-using the property `additionalTemplateFilters`.
+You may wish to extend the filters and globals with your own custom ones. For example `${{ myGlobal | myFilter | myOtherFilter }}` or `${{ myFunctionGlobal(1,2) | myFilter }}`.
```

Comment:
I'm missing an introduction to globals, would you mind adding a sentence about that?

Reply:
I believe a great way to introduce globals might be to highlight the differences to filters.
Something along the lines of "A filter generates data, whereas a filter transforms it".

Would you be ok with going with that?

Reply:
That sounds good to me 🙂